pkgs/by-name/li/libgff/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libgff";
+  version = "2.0.0";
+  src = fetchFromGitHub {
+    owner = "COMBINE-lab";
+    repo = "libgff";
+    rev = "v${version}";
+    sha256 = "sha256-ZCb3UyuB/+ykrYFQ9E5VytT65gAAULiOzIEu5IXISTc=";
+  };
+  nativeBuildInputs = [ cmake ];
+
+  meta = {
+    description = "A lightweight GTF/GFF parsers exposing a C++ interface";
+    longDescription = ''
+    This is an attempt to perform a simple "libraryfication" of the GFF/GTF
+    parsing code that is used in GFFRead codebase. There are not many (any?)
+    relatively lightweight GTF/GFF parsers exposing a C++ interface, and the
+    goal of this library is to provide this functionality without the necessity
+    of drawing in a heavy-weight dependency like SeqAn. Note: This library
+    draws directly from the code in GFFRead and GCLib, and exists primarily to
+    remove functionality (and hence code) that is unnecessary for our
+    downstream purposes. '';
```

Comment:
I probably wouldn't even add the `longDescription`, but please at least indent it.